app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -62,9 +62,13 @@ def textual_physical_servers
     h
   end
 
-  def textual_hosts
+  def textual_physical_servers_with_host
     count_of_host_relationships = (@record.physical_servers.select { |server| !server.host.nil? }).length
-    {:label => _("Hosts"), :icon => "pficon pficon-screen", :value => count_of_host_relationships}
+    h = {:label => _("Physical Servers with Host"), :icon => "pficon pficon-server", :value => count_of_host_relationships}
+    if count_of_host_relationships > 0
+      h[:link] = "/ems_physical_infra/#{@ems.id}?display=physical_servers&options=physical_servers_with_host"
```

Comment:
We are trying to generalize the `show` and related behavior in `GenericShowMixin`. This seems to be introducing yet another way to switch between different views.

Could we just use the `display=` param here? I mean generating a link such as
`/ems_physical_infra/#{@ems.id}?display=physical_servers_with_host` ?